go/client/cmd_simplefs_set_debug_level.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+)
+
+// CmdSimpleFSSetDebugLevel is the 'fs set-debug-level' command.
+type CmdSimpleFSSetDebugLevel struct {
+	libkb.Contextified
+	level string
+}
+
+// NewCmdSimpleFSSetDebugLevel creates a new cli.Command.
+func NewCmdSimpleFSSetDebugLevel(
+	cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "set-debug-level",
+		ArgumentHelp: "<integer_level>",
```

Comment:
I wonder if it'd make sense to just have this accept strings `mobile`, 'vlog1', etc.

Reply:
I didn't want to expose those to users when we ask them to run this command.  Numbers seemed better to me.